acs-integration-tests/src/test/java/com/ge/predix/acceptance/test/ACSAcceptanceIT.java

```diff
@@ -79,74 +72,51 @@
     private PolicyHelper policyHelper;
 
     @Autowired
-    private ZacTestUtil zacTestUtil;
-
-    @Autowired
-    private Environment env;
-
-    @Autowired
     private ACSRestTemplateFactory acsRestTemplateFactory;
```

Comment:
Cannot do this because UAATestUtil is not a component. Has some state for each test.